extensions/amp-addthis/validator-amp-addthis.protoascii

```diff
@@ -29,20 +29,95 @@ tags: {  # <amp-addthis>
   html_format: AMP
   tag_name: "AMP-ADDTHIS"
   requires_extension: "amp-addthis"
+  attrs: {
```

Comment:
`data-` attributes are automatically allowed in AMP. there is no need to list them here unless there are additional rules being applied, such as making one mandatory or restricting their values. please remove these.